src/contract/spec.ts

```diff
@@ -485,6 +487,22 @@ export class Spec {
     }
   }
 
+
+    /**
+     * Determines if a given method is read-only.
```

Comment:
I'm not sure this is valuable since this method should not be edited by the user; can we generate this from the specification? cc @chadoh